.github/workflows/release-copy-github-io-to-sourceforge.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Copy Site to SourceForge.
```

Comment:
Site should be changed too.

There are some other places that say `site` too. They should probably be changed or atleast reviewed.
Example:
````
  copy-site:
    name: Copy site to sourceforge for ${{ inputs.version }}
````

Reply:
Copy github.io to sourceforge.org

Reply:
Done.